### PR TITLE
Fixes in issuepage

### LINKED ISSUE #58: Write Proper CSS and make it similar to github
image

### PR DESCRIPTION
**Pull request by sridhar02**
Fixes #58

### CODE REVIEW

File: templates/issue.html [hunk 1]
```diff
@@ -283,6 +283,14 @@
 			</div>
 		</div>
 		<div>
+			<div class="participants"> 
+				<div class="number-participants">{{$.NumberOfCommented}} participant</div>
```
Comment (vramana) [inline templates/issue.html]: We should show `participants` instead of `participant` if `NumberOfCommented > 1`

File: templates/issue.html [hunk 2]
```diff
@@ -292,41 +300,29 @@
...
 					{{if lt $.PinnedIssuesCount 3}}
-					<button type="submit" > Pin Issue </button>
+					<button type="submit" ><i class="fas fa-thumbtack"></i> Pin Issue </button>
 					{{end}}
 				</form>
 				{{end}}
 				{{if eq .Pinned "Pinned"}}
 				<form method="post" action="/{{$.UserName}}/{{$.RepoName}}/issues/{{.IssueNumber}}/unpin">
 					<input name="issue_id" value="{{.ID}}" hidden></input>
 					<input name="issue_number" value="{{.IssueNumber}}" hidden></input>
```
Comment (vramana) [inline templates/issue.html]: issue number is unnecessary. Just get it from c.Params

File: templates/issue.html [hunk 3]
```diff
@@ -292,41 +300,29 @@
...
```
Comment (vramana) [inline templates/issue.html]: issue number is unnecessary. Just get it from params

File: templates/issue.html [hunk 4]
```diff
@@ -292,41 +300,29 @@
...
 					<input name="repo_id" value="{{.RepoId}}" hidden></input>
```
Comment (vramana) [inline templates/issue.html]: repo Id should be derived from get current repo instead of passed from `<form>`.

File: templates/issue.html [hunk 5]
```diff
@@ -292,41 +300,29 @@
...
-				<div class="participants-photo">
-				{{range $.CommentedUsers}}
-					<div class="user-photo"><img class="user-photo" src="{{.}}"></div>
-				{{end}}
-				</div>
-			</div>
-			{{if $.Authorized}}
-			{{if $.RepoOwner}}
-			<div>
+			<div class="lock-section">
```
Comment (vramana) [inline templates/issue.html]: class is not used anywhere.

File: templates/issue.html [hunk 6]
```diff
@@ -283,6 +283,14 @@
 			</div>
 		</div>
 		<div>
+			<div class="participants"> 
+				<div class="number-participants">{{$.NumberOfCommented}} participant</div>
+				<div class="participants-photo">
+				{{range $.CommentedUsers}}
+					<div class="user-photo"><img class="user-photo" src="{{.}}"></div>
```
Comment (vramana) [inline templates/issue.html]: - img is size too big.
- Rounded corners.

File: templates/issue.html (hunk 5 above)
Comment (sridhar02) [reply templates/issue.html]: done

File: templates/issue.html (hunk 1 above)
Comment (sridhar02) [reply templates/issue.html]: DONE

File: templates/issue.html (hunk 2 above)
Comment (sridhar02) [reply templates/issue.html]: done

File: templates/issue.html (hunk 3 above)
Comment (sridhar02) [reply templates/issue.html]: done

File: templates/issue.html (hunk 4 above)
Comment (sridhar02) [reply templates/issue.html]: done

File: templates/issue.html (hunk 6 above)
Comment (sridhar02) [reply templates/issue.html]: done

**Pull request merged**